src/app/common/data.helpers.spec.ts

```diff
@@ -336,14 +336,21 @@ it("should check description for ExecuteTransform block (no data)", () => {
 [
     { engine: "spark", label: "Apache Spark" },
     { engine: "flink", label: "Apache Flink" },
-    { engine: "datafusion", label: "DataFusion" },
-    { engine: "unknown", label: "Unknown engine" },
```

Comment:
@dmitriy-borzenko Note that we should always expect that the server is a few versions ahead of the UI and has more metadata events, engines etc. Currently UI was showing `risingwave` engine as `Unknown engine`, which was hiding the useful actual value in metadata from the user.

Reply:
In other words - for things we don't support we should be trying to propagate raw information to the user.

This will especially be important when we allow custom metadata extensions where other companies can define their own events etc.